src/Plugin/Action/UserRoleRemove.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
```

Comment:
There should be a @file comment of the beginning of each file. Please check other Action plugins for reference.